src/pages/UseCallback.mdx

```diff
@@ -0,0 +1,34 @@
+import ProperUseCallback from '../examples/hooks/properUseCallback';
+import UselessUseCallback from '../examples/hooks/uselessUseCallback';
+
+## useCallback
+
+### Use `useCallback()` hook carefully, if you have a control on you children components
```

Comment:
may be worth adding a note on this in the TMP perf guide, can you add it? Or it could be a link to this section.

Reply:
Added a link